.cloudbuild/deploy.yaml

```diff
@@ -2,19 +2,19 @@
 
 timeout: 2700s # set build timeout to 45 mins
 steps:
-  - name: node:16
+  - name: node:16.14.2
```

Comment:
My understanding is that https://hub.docker.com/_/node/ has the list of valid tags to use, including `16.14.2`. Explicitly asking for a given version instead of relying on the generic `16` seemed safer.

We obviously should double-check Cloud Build after this PR is merged to confirm.